docker-compose.yaml

```diff
@@ -12,6 +38,13 @@ services:
             DEFAULT_LOCALE:
             SERVICE_NAME:
             DATABASE_URL: pgsql://postgres:@db:5432/postgres
+            AWS_ACCESS_KEY_ID: *AWSKey
+            AWS_SECRET_ACCESS_KEY: *AWSSecret
+            AWS_DEFAULT_REGION: *AWSRegion
+            S3_ENDPOINT: http://s3:9000
+            S3_BUCKET: *S3Bucket
+            S3_PATH_STYLE_ENDPOINT: 'true'
+            ASSETS_PUBLIC_URI: http://localhost:8081/bucket
```

Comment:
would document (`README` or `.env.dist`) that this is a ingest-time configuration, so (like in Continuum) changing it requires a backfill rather than a DNS update